### PR TITLE
1.1.0

### PR DESCRIPTION
**Pull request by c-seeger**
Release Version 1.1.0

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,158 @@
...
-# Virtomize
+![Logo](https://virtomize.com/logo-text.svg)
 
-![Logo](https://github.com/Virtomize/virtomize/blob/master/logo.svg) 
 
-## Create - Deploy - Provision
+# Create - Deploy - Provision
 
-Virtomize is a software automation solution on top of your hypervisor.
+[Virtomize](https://virtomize.com) is a commercial software build for hypervisors to create, install and configure virtual machines with only a few clicks or via [REST API](https://docu.virtomize.com).
```
Comment (FrankT1983) [inline README.md]: Virtomize is a commercial software for simplifying interaction with a hypervisor, e.g. to create, install and configure virtual machines with only a few clicks or via REST API.

File: README.md [hunk 2]
```diff
@@ -1,22 +1,158 @@
...
 
-It allows you to **create, install** and **configure virtual machines** with only a few clicks.
+For currently supported hypervisors and operating systems please take a look on our [FAQ](https://virtomize.com/faq/).
 
-[![test-now](https://img.shields.io/badge/test-now-%231e828c?style=for-the-badge)](https://virtomize.com/downloads)
+We have a [free version](https://virtomize.com/pricing) just install Virtomize using this repo or download a pre-build [virtual appliance](https://virtomize.com/downloads).
```
Comment (FrankT1983) [inline README.md]: free version**. J**ust install

File: README.md [hunk 3]
```diff
@@ -1,22 +1,158 @@
...
+
+- docker
+- docker-compose
+- apache or nginx
+- git
+
+For more information please consult our [documentation page](https://docu.virtomize.com).
+Make sure the server running our software can request your hypervisor API.
+
+we use a simple docker-compose setup to make it as easy as possible.
```
Comment (FrankT1983) [inline README.md]: **W**e

File: README.md [hunk 4]
```diff
@@ -1,22 +1,158 @@
...
+# add dhparm
+curl https://ssl-config.mozilla.org/ffdhe2048.txt > /etc/nginx/ssl/dhparam
+cp /opt/virtomize/proxy/nginx.conf /etc/nginx/sites-available/default
+systemctl restart nginx
+```
+[nginx config](proxy/nginx.conf)
+
+# Update manually
+
+To update virtomize manually use the following command.
```
Comment (FrankT1983) [inline README.md]: To manually update Virtomize use the following command.

File: README.md [hunk 5]
```diff
@@ -1,22 +1,158 @@
...
+```
+cd /opt/virtomize
+./update.sh -m
+```
+
+# Backup
+
+## Default Backups
+
+After starting using docker-compose some folders are created including the `backup` folder.
```
Comment (FrankT1983) [inline README.md]: After starting **Virtomize** using docker-compose **,** some folder are created **,** including the `backup` folder.  

Maybe specify the full backup path.

File: README.md [hunk 6]
```diff
@@ -1,22 +1,158 @@
...
+We do make daily backups and keep them for 30 days.
+Just copy or sync these backups from the `backup` folder to your favorit backup storage.
```
Comment (FrankT1983) [inline README.md]: favorit**e**

File: install.sh [hunk 7]
```diff
@@ -0,0 +1,127 @@
...
+while getopts b:n flag
+do
+    case "${flag}" in
+        b) export VTMZBRANCH=${OPTARG};;
+        n) export SERVICESTART=false
+    esac
+done
+
+# https://computingforgeeks.com/install-docker-and-docker-compose-on-debian-10-buster/
+# prequisites
```
Comment (FrankT1983) [inline install.sh]: Prerequisites

File: install.sh [hunk 8]
```diff
@@ -0,0 +1,127 @@
...
+
+# docker repo
+curl -fsSL https://download.docker.com/linux/debian/gpg | apt-key add -
+add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/debian $(lsb_release -cs) stable"
+
+# install docker
+apt update
+apt -y install docker-ce docker-ce-cli containerd.io
+
+# install docker
```
Comment (FrankT1983) [inline install.sh]: duplicate comment to line 37

File: README.md (hunk 1 above)
Comment (c-seeger) [reply README.md]: Hm how do you like the new one?

**Pull request merged**